prov/usnic/src/debugme_func.c

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
I assume that this whole file is just for debugging and will not be included in the final PR.